### PR TITLE
pre-alpha vision of product page

### PR DESCRIPTION
**Pull request by Clover247**
https://youtu.be/uxndeoVCx4k
For current control only. The page will be divided into components, the layout will be changed, and the request will be made via a link.

### CODE REVIEW

File: src/modules/PhonePage/PhonePage.tsx [hunk 1]
```diff
@@ -1,6 +1,253 @@
...
+          <p className={styles.aboutSectionText}>{description[0].text[1]}</p>
+        </section>
+        <section className={styles.aboutSection}>
+          <h3 className={styles.aboutSectionHeader}>{description[1].title}</h3>
+          <p className={styles.aboutSectionText}>{description[1].text}</p>
+        </section>
+        <section className={styles.aboutSection}>
+          <h3 className={styles.aboutSectionHeader}>{description[2].title}</h3>
+          <p className={styles.aboutSectionText}>{description[2].text}</p>
+        </section>
```
Comment (BetterSol) [inline src/modules/PhonePage/PhonePage.tsx:206]: maybe use `.map` for this?

File: src/modules/PhonePage/PhonePage.tsx [hunk 2]
```diff
@@ -1,6 +1,253 @@
...
+          <li className={styles.techInfoParametr}>
+            <h3 className={styles.techSpecsKey}>Zoom</h3>
+            <h3 className={styles.techSpecsValue}>{zoom}</h3>
+          </li>
+          <li className={styles.techInfoParametr}>
+            <h3 className={styles.techSpecsKey}>Cell</h3>
+            <h3 className={styles.techSpecsValue}>{cell.join(', ')}</h3>
+          </li>
+        </ul>
+      </article>
```
Comment (BetterSol) [inline src/modules/PhonePage/PhonePage.tsx:246]: here too

**Review by BetterSol — COMMENTED** [review]
Awesome 🔥

**Review by BetterSol — APPROVED** [review]

**Comment by Clover247** [comment]
https://youtu.be/ceZU-yRcAYk not final too(

**Pull request closed without merging**